__tests__/app.test.js

```diff
@@ -378,81 +377,125 @@ describe("PATCH /api/articles/:article_id", () => {
-    })
-  })
+      .patch("/api/articles/1")
+      .send(newVotes)
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe(
+          `400 - Bad request, must enter inc_votes: Number`
+        );
+      });
+  });
+});
+
+describe("DELETE /api/comments/:comment_id", () => {
+  test("204: Responds with 204 and deletes given comment on the comment_id and returns no content back", () => {
+    return request(app).delete("/api/comments/1").expect(204);
+  });
+
+  test("404: Responds with 404 not found if comment with that ID doesn't exist before deletion", () => {
+    return request(app)
+      .delete("/api/comments/99999")
+      .expect(404)
+      .then((res) => {
+        expect(res.body.msg).toBe(
+          `404 - Not found, that comment doesn't exist`
+        );
+      });
+  });
+
+  test("400: Responds with 400 bad request if comment_id given is not an umber", () => {
+    return request(app)
+      .delete("/api/comments/banana")
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe(
+          `400 - Bad request, comment_id must be a number`
+        );
+      });
+  });
 });
 
-describe("DELETE /api/comments/:comment_id",()=>{
-  test("204: Responds with 204 and deletes given comment on the comment_id and returns no content back",()=>{
+describe("GET /api/users", () => {
+  test("200: Will get all users", () => {
     return request(app)
-    .delete("/api/comments/1")
-    .expect(204)
-  })
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toHaveLength(4);
+        body.forEach((user) => {
+          expect(user).toMatchObject({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          });
+        });
+      });
+  });
 
-  test("404: Responds with 404 not found if comment with that ID doesn't exist before deletion",()=>{
+  test("GET:404 responds with 404 not found", () => {
     return request(app)
-    .delete("/api/comments/99999")
-    .expect(404)
-    .then((res)=>{
-      expect(res.body.msg).toBe(`404 - Not found, that comment doesn't exist`)
-    })
-  })
-
-  test("400: Responds with 400 bad request if comment_id given is not an umber",()=>{
+      .get("/api/endpoint-that-doesnt-exist")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("404 - Not found");
+      });
+  });
+
+  test("GET:400 responds with 400 when given invalid params", () => {
     return request(app)
-    .delete("/api/comments/banana")
-    .expect(400)
-    .then((res)=>{
-      expect(res.body.msg).toBe(`400 - Bad request, comment_id must be a number`)
-    })
-  })
-
-  
-})
\ No newline at end of file
+      .get("/api/users?nonexistantParam=Nope")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("400 - Bad request, invalid user parameters");
+      });
```

Comment:
I don't think this test is necessary, unless you had happy path tests that dealt with queries too. There is an advanced task that asks you to get a specific user but that would use the endpoint /api/users/:username - so a parametric endpoint rather than a query. 

For this specific task, there's nothing that can really go wrong for /api/users so no error testing needed